### ISSUE TITLE
Allow API called by glossary modals to be configurable

### ISSUE THREAD

**Issue by blairlearn** [NCIOCPL/cgov-digital-platform#2663]
### Issue description
The code to display pop-up dictionary terms hard-codes the use of 'https://www.cancer.gov/Dictionary.Service/v1' as the endpoint for looking up the term definition. This prevents integration testing in the event of changes to the service (e.g. a bug in the JSON rendering).

It would be helpful if the code instead used a value from the front end globals object (e.g. `window.CDEConfig.general.apiServer`)

> **ESTIMATE** 2

### What's the expected change?
- Glossary term modal popups can be configured to use non-production endpoints.

### What's the current functionality?
- Glossary term modal popups always use `https://www.cancer.gov/Dictionary.Service/v1`

### What's the updated acceptance criteria?
```gherkin
Scenario: Display a glossary term in a modal popup
  Given An administrator has changed the value of window.CDEConfig.general.apiServer to a non-production endpoint
  When  A user displays a glossary term in a modal popup
  Then  The term is looked up via the non-production endpoint.
```

### Additional details
The endpoint is currently hard-coded in:  [docroot/profiles/custom/cgov_site/themes/custom/cgov/src/libraries/dictionaryService/index.js](https://github.com/NCIOCPL/cgov-digital-platform/blob/master/docroot/profiles/custom/cgov_site/themes/custom/cgov/src/libraries/dictionaryService/index.js#L37)

**Comment by blairlearn** [comment]
This should be done *after* the GlossaryAPI is released, rather than modifying once for the old API and again for the new one.

**Comment by bryanpizzillo** [comment]
This appears to have been completed as part of the fix for #2789 in PR #2797.
